quinn-proto/Cargo.toml

```diff
@@ -11,7 +11,7 @@ categories.workspace = true
 workspace = ".."
 
 [features]
-default = ["rustls-ring", "log"]
+default = ["rustls-ring", "log", "fastbloom"]
```

Comment:
What is the argument for enabling this by default?

Reply:
Okay, I think I can live with it being enabled by default. Thanks for the context!